routes/api/books.js

```diff
@@ -22,7 +22,10 @@ router.post('/', (req, res) => {
     // NOTE: youtuber video at 27min
     name: req.body.name,
     author: req.body.author,
-    category: req.body.category
+    category: req.body.category,
```

Comment:
If you are going to use only `req.body` object, you can use object destructuring and object property value shorthand here. For example, with one property:
```
router.post('/', ({ body }, res) => {
  const { name } = body
  const newBook = new Book({
    name,
  })
})
```

More info in:
[Object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)
[Property Shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions)